ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala

```diff
@@ -4,15 +4,20 @@
 package com.daml.metrics
 
 import com.daml.metrics.api.MetricDoc.MetricQualification.{Debug, Saturation, Traffic}
-import com.daml.metrics.api.MetricHandle.{Counter, Factory, Histogram, Timer}
+import com.daml.metrics.api.MetricHandle.{
+  Counter,
+  Histogram,
+  LabeledMetricsFactory,
+  MetricsFactory,
+  Timer,
+}
 import com.daml.metrics.api.dropwizard.DropwizardTimer
-import com.daml.metrics.api.opentelemetry.OpenTelemetryFactory
 import com.daml.metrics.api.{MetricDoc, MetricName}
 
 class ServicesMetrics(
     prefix: MetricName,
-    factory: Factory,
-    openTelemetryFactory: OpenTelemetryFactory,
+    factory: MetricsFactory,
+    openTelemetryFactory: LabeledMetricsFactory,
```

Comment:
`openTelemetryFactory` should be renamed to `labelMetricsFactory`, like it has been done in all other locations.